packages/web/src/common/store/notifications/parseAndProcessNotifications.ts

```diff
@@ -85,58 +85,65 @@ export function* parseAndProcessNotifications(
       }
     }
     if (
-      notification.type === NotificationType.Follow ||
-      notification.type === NotificationType.Repost ||
-      notification.type === NotificationType.Favorite
+      type === NotificationType.Follow ||
+      type === NotificationType.Repost ||
+      type === NotificationType.RepostOfRepost ||
+      type === NotificationType.Favorite ||
+      type === NotificationType.FavoriteOfRepost
     ) {
-      // @ts-ignore
       notification.userIds = [...new Set(notification.userIds)]
       userIdsToFetch.push(
         ...notification.userIds.slice(0, USER_INITIAL_LOAD_COUNT)
       )
     }
-    if (notification.type === NotificationType.RemixCreate) {
+    if (type === NotificationType.RemixCreate) {
       trackIdsToFetch.push(
         notification.parentTrackId,
         notification.childTrackId
       )
       notification.entityType = Entity.Track
-      notification.entityIds = [
```

Comment:
just confirming this is no longer needed?

Reply:
yes! it was a mistyped notif